File: baekjoon/1110.js
const readline = require('readline')
const rl = readline.createInterface({
  input: process.stdin,
  output: process.stdout,
})

const handleInputLine = (line) => {
  let N = Number(line)
  let i = 0,
    sum = 0
  while (true) {
    i++
    sum = Math.floor(N / 10) + (N % 10)
    N = (N % 10) * 10 + (sum % 10)
    if (N === Number(line)) break
  }
  console.log(i)
}
const handleOutputLine = () => {
  process.exit()
}

rl.on('line', handleInputLine).on('close', handleOutputLine)

File: baekjoon/while문/1110.py
N = int(input())
count = 1
newN = 0

def getN(number):
  A = 0
  B = 0
  C = 0
  if len(f'{number}') > 1:
    A = int(f'{number}'[0])
    B = int(f'{number}'[1])
  else:
    A = int(f'{number}'[0])
    B = 0
  C = A + B
  return int(str(number % 10) + str(C % 10))

newN = getN(N)
while newN != N:
  count = count+1
  newN = getN(newN)
print(count)